### PR TITLE
Fix plugin export

### PR DESCRIPTION
**Pull request by vinokurig**
`pluginMain[plugin.lifecycle.startMethod].apply(getGlobal(), [pluginContext])` returns a `Promise`, so need to resolve it.
Is needed for https://github.com/eclipse/che/issues/11867

### CODE REVIEW

**Comment by akosyakov** [comment]
@vinokurig How can one verify it in Theia repo?

**Comment by vinokurig** [comment]
I've tested on Vscode github pull request plugin in pair with vscode Git Extension

File: packages/plugin-ext/src/plugin/plugin-manager.ts [hunk 1]
```diff
@@ -171,7 +171,10 @@ export class PluginManagerExtImpl implements PluginManagerExt, PluginManager {
             stopFn = pluginMain[plugin.lifecycle.stopMethod];
         }
         if (typeof pluginMain[plugin.lifecycle.startMethod] === 'function') {
-            const pluginExport = pluginMain[plugin.lifecycle.startMethod].apply(getGlobal(), [pluginContext]);
+            let pluginExport = pluginMain[plugin.lifecycle.startMethod].apply(getGlobal(), [pluginContext]);
+            if (pluginExport instanceof Promise) {
```
Comment (akosyakov) [inline packages/plugin-ext/src/plugin/plugin-manager.ts]: `instanceof` is unnecessary, you can `await` on anything

Comment (vinokurig) [reply packages/plugin-ext/src/plugin/plugin-manager.ts]: fixed

**Comment by vinokurig** [comment]
@evidolob @benoitf Any updates?

**Pull request merged**